location.go

```diff
@@ -35,3 +40,23 @@ func DefaultWindowLocation() Location {
 		Longitude: 172.6362,
 	}
 }
+
+func locationToMap(f reflect.Type, t reflect.Type, data interface{}) (interface{}, error) {
+	if t != reflect.TypeOf(map[string]interface{}{}) {
```

Comment:
if `reflect.TypeOf(map[string]interface{}{})` gets used to a lot in go-config it's probably worth making it a module level variable.